QMLComponents/analysisbase.cpp

```diff
@@ -4,6 +4,7 @@
 #include "utilities/qmlutils.h"
 
 const std::string AnalysisBase::emptyString;
+const stringvec AnalysisBase::emptyStringVec;
```

Comment:
`stringvec upgradeMsgsForOption(std::string name)` used to return `{""}` not `{}`...

```suggestion
const stringvec AnalysisBase::emptyStringVec = {""};
```

Reply:
Aah ok, but that seems to be caused by the move to AnalysisBase and wasnt originally like that at all